styleguide/_themes/derek/scss/components/tabs.scss

```diff
@@ -4,8 +4,8 @@
   text-align: center;
 
   & > li {
-    background-color: $darkteal;
-    border-top: 5px solid $darkteal;
+    background-color: $dark-teal;
```

Comment:
Would we want to use `teal-dark` to stick with the convention set for gray (e.g. `gray-light`)?